scripts/sts/sts-secret.yaml

```diff
@@ -1,8 +0,0 @@
-apiVersion: v1
```

Comment:
Think we should keep this to aid local development as STS support is technically RHOAM agnostic, and I don't think it's applied as part of https://github.com/integr8ly/delorean/pull/254/files :thinking: